docs/server/source/simple-network-setup.md

```diff
@@ -60,15 +60,17 @@ sudo pip3 install bigchaindb==2.0.0a6
 
 Check that you installed the correct version of BigchainDB Server using `bigchaindb --version`.
 
-#### Install MongoDB
+#### Install (and Start) MongoDB
 
 Install a recent version of MongoDB. BigchainDB Server requires version 3.4 or newer.
 
 ```
 sudo apt install mongodb
 ```
 
-The package manager should take care of installing the startup script for MongoDB as well. (That means MongoDB will automatically start if the machine is restarted.)
+If you install MongoDB using the above command (which installs the `mongodb` package), it also configures MongoDB, starts MongoDB (in the background), and installs a MongoDB startup script (so that MongoDB will be started automatically when the machine is restarted).
+
+Note: The `mongodb` package is _not_ the official MongoDB package from MongoDB the company. If you want to install the official MongoDB package, please see [the MongoDB documentation](https://docs.mongodb.com/manual/installation/). Note that installing the official package _doesn't_ also start MongoDB.
```

Comment:
I guess you added this paragraph because of the notes on the [MongoDB installation guide](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/#overview). Every GNU/Linux distribution has its maintainers that are not necessarily connected to the original project. I'm wondering what MongoDB is trying to communicate there, because to me this sounds a bit like FUD. Ideas?

Reply:
I probably read that paragraph in the MongoDB installation guide in the past, but I was also writing from experience. In the past, e.g. in our Quickstart page, we always told people to install the official MongoDB package from MongoDB the company, and there were extra instructions about how to create the data directory and how to start MongoDB. That's also what this other page _still_ says to do:

https://docs.bigchaindb.com/projects/contributing/en/latest/dev-setup-coding-and-contribution-process/run-node-as-processes.html#install-and-run-mongodb

As for the motives of MongoDB, I suspect they don't want to have to support MongoDB packages made by outsiders. That seems reasonable.